docs/modules/ROOT/partials/storage-ceph-cleanup-osd.adoc

```diff
@@ -1,40 +1,54 @@
-. Make a note of the PVC of the old OSD
+. Make a note of the PVCs of the old OSDs
 +
-NOTE: We also extract the name of the PV here, but we'll only delete the PV after removing the node from the cluster.
+NOTE: We also extract the name of the PVs here, but we'll only delete the PVs after removing the nodes from the cluster.
 +
-[source,bash]
-----
-pvc_name=$(kubectl --as=cluster-admin -n syn-rook-ceph-cluster get deploy \
-  "rook-ceph-osd-${OSD_ID}" -ojsonpath='{.metadata.labels.ceph\.rook\.io/pvc}')
-pv_name=$(kubectl --as=cluster-admin -n syn-rook-ceph-cluster get pvc \
-  "${pvc_name}" -o jsonpath='{.spec.volumeName}')
+[source,bash,subs="attributes+"]
+----
+old_pvc_names=""
+old_pv_names=""
```

Comment:
you probably should use the page attribute `:delete-pvs:` here to ensure the partials stay consistent with each other for any value of `:delete-pvs:`.